### DISCUSSION TITLE
How to change the background color from the title of the notifier?

### DISCUSSION THREAD

**Discussion by Raagh** [LazyVim/LazyVim#5156] [Q&A]
Hi I am trying to change the background color for the title of the notification history and others, I think probably is the snacks.nvim plugin but I am not sure and couldn't find a way to change it.

I managed to change it for fzf-lua this is how it looks
![grep](https://github.com/user-attachments/assets/2080a900-9581-47b1-a91f-3add824769c3)

but this is how it looks for the scratch buffer
![scratch](https://github.com/user-attachments/assets/c9571059-72f8-4cad-b9f4-bc06cf8ceca5)

**Comment by Raagh** [comment]
@dpetka2001 hey sorry to ping you but you are usually my savior when it comes to this issues, do you have any idea?

**Reply by dpetka2001** [reply]
No idea sorry. I don't change much the colors except the colorscheme defaults.
